scripts/bld.bat

```diff
@@ -16,11 +16,11 @@ rem limitations under the License.
 rem ============================================================================
 
 set DAAL4PY_VERSION=%PKG_VERSION%
-set MPIROOT=%PREFIX%\Library
+set MPIROOT=%CONDA_PREFIX%\Library
```

Comment:
Why this is still a `PREFIX` for linux then?

Reply:
thx, fixed